articles/search/search-security-overview.md

```diff
@@ -166,17 +166,18 @@ Currently, the only external resource that a search service writes customer data
 
 ### Exceptions to data residency commitments
 
-Although customer data isn't stored outside of your region, object names will appear in the telemetry logs used by Microsoft Support to troubleshoot your service issues. Object names are considered customer data. Names in telemetry logs include those of indexes, indexers, data sources, skillsets, containers, and key vault store.
+Although customer data isn't stored outside of your region, object names (considered as customer data), will appear in the telemetry logs used by Microsoft Support to troubleshoot your service issues. Telemetry logs includes names of indexes, indexers, data sources, skillsets, containers, and key vault store.
```

Comment:
Telemetry logs include (no "s")

Reply:
Hi Angie -- this isn't resolved.  We have a noun-verb issue here -- can you change this sentence to be "Telemetry logs include the names of indexes, indexers, data sources, skillsets, containers and key vault stores."